x-pack/plugins/security_solution/public/common/utils/route/spy_routes.tsx

```diff
@@ -57,7 +57,7 @@ export const SpyRouteComponent = memo<
               pathName: pathname,
               state,
               tabName,
-              ...(pageName === SecurityPageName.administration ? { search: search ?? '' } : {}),
+              search: search ?? '',
```

Comment:
Why is this dispatch of _updateRouteWithOutSearch_ action having the `search` prop? Shouldn't we remove the `search` prop from this action payload, as we were doing before for all the pages except administration?

Reply:
Oops, nice catch!